### ISSUE TITLE
web remote invalid user

### ISSUE THREAD

**Issue by lukasa1993** [BiglySoftware/BiglyBT#811]
* OS and version osx mojave
* BiglyBT Version Number latest for mojave
* For *nix: Desktop Environment

it was working perfectly and today all of a sudden it says incorrect user name when its same exactly same, can it print what it received wrong or some other way to debug this?

**Comment by parg** [comment]
BiglyBT version 1.7.0.0? Where exactly are you getting the 'invalid user name' reported?

**Comment by lukasa1993** [comment]
yes 1.7.0, i am getting it in web remote log

**Comment by parg** [comment]
If you are using the 'access code' to login the the user name should be pre-configured to be 'vuze' and the password will be the access code.

Check you remote webui configuration in case something changed

**Comment by lukasa1993** [comment]
whats access code? i am using basic RPC nothing else

**Comment by lukasa1993** [comment]
if you mean stuff under parring section its disabled

**Comment by parg** [comment]
So in the log it says

Access denied: Incorrect user name: <name>

and <name> is exactly what is configured?

**Comment by lukasa1993** [comment]
```Access denied: Incorrect user name: ``` this is log message

**Comment by parg** [comment]
So it is receiving a blank username - sounds like your 'Authorization:' header is missing

**Comment by lukasa1993** [comment]
hmm interesting, can it print out all headers received ?

**Comment by parg** [comment]
so it was working and now it isn't but absolutely nothing has changed?

**Comment by lukasa1993** [comment]
yes something changed ofc but i cant find what, i certainly haven't changed anything

**Comment by parg** [comment]
How are you invoking the RPC?

**Comment by lukasa1993** [comment]
https://github.com/alcheck/transshift

with this

and response seems to be this:
```CredStore - performQuery - Error copying matching creds.  Error=-25300, query={
    class = inet;
    "m_Limit" = "m_LimitAll";
    ptcl = htps;
    "r_Attributes" = 1;
    sdmn = "BiglyBT Extreme Mod - BiglyBT Web Remote";
    srvr = "dasdasda";
    sync = syna;
}```

**Comment by parg** [comment]
wireshark?

**Comment by lukasa1993** [comment]
> wireshark?

its throe https, i removed auth all to gather :)

**Comment by garfield69** [comment]
@lukasa1993  I ran into this today.
I could not add magnets via my app's RPC to BiglyBT, and so too the Chrome Browser WEBUI to BiglyBT did not allow me to add magnets, both failing with the same error (Access denied: Incorrect user name).

After some research I discovered that when I upgraded BiglyBT 1.8.0 to 2.1.0, 
the **BiglyBT Web Remote** plugin setting for *Access Control Mode* was set to **View** instead of the default **full**.
Once I change it back to **full**, saved and restarted the apps, my app RPC was working again and so too the Chrome Browser WEBUI.
